src/containers/PermissionsApplication.spec.jsx

```diff
@@ -50,6 +50,14 @@ jest.mock('cozy-ui/transpiled/react/Spinner', () => {
   // eslint-disable-next-line react/display-name
   return ({ size }) => <div data-testid="Spinner" data-size={size}></div>
 })
+jest.mock('cozy-ui/transpiled/react', () => {
+  const t = text => text
+  return {
+    useI18n: () => ({
+      t
+    })
+  }
+})
```

Comment:
shorten this line:

jest.mock('cozy-ui/transpiled/react', () => ({
  useI18n: () => ({ t: text => text })
}))